libmscore/textbase.cpp

```diff
@@ -528,13 +528,16 @@ void TextFragment::draw(QPainter* p, const TextBase* t) const
             // in Qt 5.12.x this workaround should be no more necessary if
             // env variable QT_MAX_CACHED_GLYPH_SIZE is set to 1
             p->save();
+            int deviceLogicalDpi = p->device()->logicalDpiX();
+            QScreen* screen = QGuiApplication::primaryScreen();
+            qreal screenDpi = screen->logicalDotsPerInch();
```

Comment:
`logicalDotsPerInch()` gets you the averag of `logicalDotsPerInchX()`and `logicalDotsPerInchY()`, maybe better do the same above for `deviceLogicalDpi`? Like `qreal deviceLogicalDpi = (p->device()->logicalDpiX() + p->device()->logicalDpiY()) / 2.0;`